### PR TITLE
Fixing backticks that were rendering

### PR DESCRIPTION
**Pull request by bergerhoffer**

### CODE REVIEW

File: _topic_map.yml [hunk 1]
```diff
@@ -1345,7 +1345,7 @@ Topics:
     File: preparing-cluster-for-cnv
   - Name: Installing container-native virtualization
     File: installing-container-native-virtualization
-  - Name: Installing the `virtctl` client
+  - Name: Installing the virtctl client
```
Comment (bergerhoffer) [inline _topic_map.yml]: @ousleyp @aburdenthehand Backticks don't get replaced properly in the TOC and they're rendering, so I'm removing. Let me know if you have any concerns with that, thanks!

Comment (ousleyp) [reply _topic_map.yml]: Good catch, thanks!

File: modules/nodes-cluster-resource-configure-jdk.adoc [hunk 2]
```diff
@@ -91,13 +91,13 @@ that the right settings are being passed to the right JVM.
 
 The `JAVA_TOOL_OPTIONS` environment variable is always respected by the OpenJDK,
 and values specified in `JAVA_TOOL_OPTIONS` will be overridden by other options
-specified on the JVM command line. By default,  to ensure that these options are 
+specified on the JVM command line. By default,  to ensure that these options are
```
Comment (jboxman) [inline modules/nodes-cluster-resource-configure-jdk.adoc]: _By default,  to_ extra white space

**Review by jboxman — APPROVED** [review]
Did find one double spacing between two words, but otherwise LGTM.

File: modules/nodes-cluster-resource-configure-jdk.adoc (hunk 2 above)
Comment (bergerhoffer) [reply modules/nodes-cluster-resource-configure-jdk.adoc]: Sure, happy to fix that as well.

**Pull request merged**